<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RapportsView</class>
 <widget class="QMainWindow" name="RapportsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Treasurer - Rapports</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background: #f5f5f5;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #000000;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>50</number>
           </property>
           <property name="leftMargin">
            <number>15</number>
           </property>
           <item>
            <widget class="QPushButton" name="comptes_button">
             <property name="font">
              <font>
               <family>Raleway</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #1556C7;</string>
             </property>
             <property name="text">
              <string>COMPTES</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="transaction_button">
             <property name="font">
              <font>
               <family>Raleway</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #ffffff;</string>
             </property>
             <property name="text">
              <string>TRANSACTIONS</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="rapports_button">
             <property name="font">
              <font>
               <family>Raleway</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #ffffff;</string>
             </property>
             <property name="text">
              <string>RAPPORTS</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cloture_button">
             <property name="font">
              <font>
               <family>Raleway</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #ffffff;</string>
             </property>
             <property name="text">
              <string>CLÃ”TURE</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>15</number>
           </property>
           <item>
            <widget class="QPushButton" name="settings_button">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../ressources/settings.svg</normaloff>../ressources/settings.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>28</width>
               <height>30</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="logout_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../ressources/quit.svg</normaloff>../ressources/quit.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QWidget" name="bilan" native="true">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>250</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>250</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>SimSun-ExtB</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#recettes {
border-style : solid; 
border-width: 1px;
border-color: rgb(56, 56, 56);
border-radius: 10px;
}

QWidget{
background: #ffffff;
}</string>
          </property>
          <widget class="QWidget" name="widget_9" native="true">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>90</y>
             <width>150</width>
             <height>2</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>2</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>2</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">.QWidget {
background: #000000;
border: none;
border-radius: 0px;
}</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_14">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>331</width>
             <height>39</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Raleway</family>
             <pointsize>27</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #1556C7;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Bilan&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QWidget" name="horizontalLayoutWidget_6">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>190</y>
             <width>311</width>
             <height>33</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPushButton" name="detail_bilan_button">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>31</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Raleway</family>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
background : #1556C7;
color : #FFFFFF;
font-weight: bold;
border-radius: 5px;
border: none;
}</string>
              </property>
              <property name="text">
               <string>DÃ‰TAIL</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="horizontalLayoutWidget_10">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>331</width>
             <height>80</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="QLabel" name="label_15">
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>12</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Actifs - Passifs :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="bilanLabel">
                <property name="font">
                 <font>
                  <family>Raleway</family>
                  <pointsize>14</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>0â‚¬</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="compteDeResultat" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>250</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>250</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>SimSun-ExtB</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#depenses {
border-style : solid; 
border-width: 1px;
border-color: rgb(56, 56, 56);
border-radius: 10px;
}

QWidget{
background: #ffffff;
}</string>
          </property>
          <widget class="QWidget" name="widget_11" native="true">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>90</y>
             <width>150</width>
             <height>2</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>2</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>2</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">.QWidget {
background: #000000;
border: none;
border-radius: 0px;
}</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_20">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>331</width>
             <height>39</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Raleway</family>
             <pointsize>27</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #1556C7;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Compte de rÃ©sultat&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QWidget" name="horizontalLayoutWidget_8">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>190</y>
             <width>311</width>
             <height>33</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QPushButton" name="detail_compteresultat_button">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>31</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Raleway</family>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
background : #1556C7;
color : #FFFFFF;
font-weight: bold;
border-radius: 5px;
border: none;
}</string>
              </property>
              <property name="text">
               <string>DÃ‰TAIL</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="horizontalLayoutWidget_12">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>331</width>
             <height>80</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_18">
              <item>
               <widget class="QLabel" name="beneficeOuPerteLable">
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>12</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>     BÃ©nÃ©fice:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="compteDeResultatLabel">
                <property name="font">
                 <font>
                  <family>Raleway</family>
                  <pointsize>14</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>0â‚¬</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
